generators/app/templates/_package.json

```diff
@@ -38,6 +38,9 @@
     "@angular/platform-browser": "^5.2.5",
     "@angular/platform-browser-dynamic": "^5.2.5",
     "@angular/router": "^5.2.5",
+<% if (props.ui === 'bootstrap') { -%>
```

Comment:
It could be grouped with the bootstrap block below :)

Reply:
I intentionally put it there to have dependencies ordered by name (alphabetically), should I move it ?

Reply:
Yes, I would prefer to group branches/conditions for maintainability, also because npm will reorder things magically anyway 😉